index.html

```diff
@@ -20,9 +20,21 @@ <h1 class="main-logo">
                 </h1>
                 <div class="worker-container">        
                     <img src="./images/mobile.png" class="mobile" alt="">
+                    <ul>
+                        <li><h3>テキストテキスト<br>テキストテキストテキスト</h3></li>
```

Comment:
MUST
hタグは順番に使うべきだと思います。
```suggestion
                        <li><h2>テキストテキスト<br>テキストテキストテキスト</h2></li>
```

Reply:
ありがとうございます！
修正しました！

e79ded7

Reply:
※二か所あるうちの一か所しか修正できていなかったです…。
　こちらで修正しました。
　
　4881e10